openssl-dynamic/pom.xml

```diff
@@ -107,6 +107,10 @@
           <family>windows</family>
         </os>
       </activation>
+      <properties>
+        <!-- Skip tests on windows as its not easy to load the openssl library we link against in the tests -->
+        <skipTests>true</skipTests>
```

Comment:
@Scottmitch there may be a way to fix this, but I am a bit short on time and would love to merge this first. We can do a followup if needed